SU-HSDO/suhumsci
# STN-781: Update caption/credit styles on images placed within the body area of node detail pages

# READY FOR REVIEW

## Summary
The work in this PR updates the caption/credit styles of images placed within the body text area of node detail pages such as an event, news item, publication, or person detail page. The caption/credit styles should match those currently used on other images through the humsci basic theme which feature a black gradient background absolutely positioned on top of the image with small white text.

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. In your CLI, run `npm run test` and confirm our tests continue to pass.
2. Go to a node detail page with an image within the body area of the page. If you have synced with staging you can use: http://sparkbox-sandbox.suhumsci.loc/news/broken-captioncredit-ct-page-0
3. - [x] Confirm the `hb-caption-credit` mixin styles that feature the black gradient background overlaying the image with white text display.
4. You can also add images to the following node detail pages to confirm that this update works for event, event series, news item, person, publication, and research area node detail pages.
    - [x] Event: http://sparkbox-sandbox.suhumsci.loc/events/1st-year-economics-graduate-student-seminar-series/qa-events-all-fields-filled
    - [x] Event Series: http://sparkbox-sandbox.suhumsci.loc/compact-laser-driven-ions-beams-combining-global-leadership-high-power-lasers-france-and-us-make
    - [x] News Item: http://sparkbox-sandbox.suhumsci.loc/news/qa-joeys-test-news-item
    - [x] Person: http://sparkbox-sandbox.suhumsci.loc/people/qa-person-all-fields-filled
    - [ ] Publication: http://sparkbox-sandbox.suhumsci.loc/publications/qa-publications-all-fields-filled
5. Additional pages to test:
    - [ ] Confirm caption/credits applied in text area components continue to look the same: http://sparkbox-sandbox.suhumsci.loc/qa/qa-text-area
6. - [ ] Confirm that images added via the body text area and images added via an image field both receive the same caption/credit styling.
    
#### Screenshot of the different ways to add images to a node detail page
<img width="742" alt="details" src="https://user-images.githubusercontent.com/12678977/125311897-c7b60780-e301-11eb-8aed-9a7e629be3a8.png">

#### Expected caption / credits results 
<img width="1059" alt="caption credits" src="https://user-images.githubusercontent.com/12678977/125311915-cbe22500-e301-11eb-8661-11de5aeb8be9.png">
